### ISSUE TITLE
Mod Packs

### ISSUE THREAD

**Issue by TheSillyDoggo** [geode-sdk/geode#363]
### Geode Version

future

### Your Solution

Add a tab to the geode menu that has a list where mod developers can add lists. It can be stuff from a mod pack (like a pack that has remakes of stuff like Extra Icons or Gradient Profiles) or just a pack for betteredit that contains Better edit, Editor api and keybinds. These packs can be updated at any time to add new mods.

I made some concept images in photoshop
![Mod Packs Concept 1](https://github.com/geode-sdk/geode/assets/95105680/f1171736-1d7c-45df-a5bf-8df81a7ac449)
![Mod Packs Concept 2](https://github.com/geode-sdk/geode/assets/95105680/493c0a5b-caa2-43fa-b9d5-c524c4f8ff21)

**Comment by Fleeym** [comment]
I'll do you one better, lists can be created by any user, using mods that are from the index

**Comment by TheSillyDoggo** [comment]
Yeah but make ones by mod developers show at the top of the list so that they appear before others cuz they would probably be better

**Comment by CattoDev** [comment]
I like this idea

**Comment by quisaw** [comment]
Maybe make a system where developers can make a repository for their mods on GitHub & users can add the repository link to a list tab. Users would be able to pick and choose what mods they want to install and enable/ disable. Think of it kind of like IOS jailbreaking ig
